docker/dev/Dockerfile.django

```diff
@@ -1,12 +1,6 @@
-# We use the official Python images. This may need to get locked in, or back to Ubuntu at some point
 FROM python:3
 
-RUN apt-get update
```

Comment:
Hmm, why did you remove the package installs?

Reply:
The reason for the package installs was just for XSLT and related.

Otherwise, the Docker image software should be maintained by the upsteam Docker maintainers. There are three reasons why this is the case:

1. This is the Docker data model

What I mean is that Docker images will only fetch data that's necessary to overlay on top of each other, recursively. It's more space efficient to do this

2. We're using the official Python docker images

Just as we would generally rely on upstream package maintainers in Linux or BSD to maintain upstream code, we should be leaning on the Python docker maintainers to maintain upstream docker. The only reason we pulled in additional packages was for XSLT, and as you mentioned, that's no longer necessary.

3. If the official Python image changes somehow, we're less likely to break.

It appears that Python's docker image is using Debian, but if it should change somehow, our docker images will still build properly if we aren't relying on anything other than Python itself.

That's why I removed it.

Reply:
Actually it was a question:

> I'm kind of surprised libxml2-dev libxslt-dev are not needed. Those are direct dependencies for lxml to be built successfully. Maybe the Python3 image has them already?

https://github.com/jaywink/socialhome/pull/334#discussion_r145684629

At least Ubuntu requires a few extra libraries to build `lxml` (see http://lxml.de/installation.html#requirements and https://socialhome.readthedocs.io/en/latest/install_guides.html#install-system-packages (federation ie lxml dependencies)). So if the `python:3` image does work without these extra installs, it must have them already? Otherwise I can't imagine that the docker images work.

Does a clean build of this docker image without these extra installs actually work? And tests also pass?